smart-contracts/contracts/PublicLock.sol

```diff
@@ -156,6 +156,7 @@ contract PublicLock is ILockCore, ERC165, IERC721, IERC721Receiver, Ownable {
     uint _maxNumberOfKeys
   )
   public {
+    require(_expirationDuration < 100 * 365 * 24 * 60 * 60, "Expiration duration must be less than 100 years");
```

Comment:
Minor, but maybe we could use <= here? Not too consequential either way.

Reply:
done